services/testTopics/index.js

```diff
@@ -38,9 +38,22 @@ function remove(id) {
     });
 }
 
+function updateTheory(id, theory) {
+    return new Promise((resolve, reject) => {
+        TestTopic.update({ theory }, { where: { id } })
+            .then(result => {
+                TestTopic.findOne({ where: { id } })
+                    .then( topic => resolve(topic))
+                    .catch( err => reject({ status: 500 }));
```

Comment:
so,you can not change smth in this project, but we need ti analyze exception object in catch block